docs/content/developers/deploy_with_helm.md

```diff
@@ -0,0 +1,84 @@
+# Deploying onos micro-services with HELM
+
+One of the goals of the onos-config project is to provide simple deployment options
+that integrate with modern technologies. Deployment configurations can be found in
```

Comment:
it would be better to replace onos-config with micro-onos

Reply:
done